presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -151,3 +151,161 @@ Hive, this table can be described in Presto::
++                                                +                                          +                                           +``orc_stream_buffer_size`` session properties defining max    +
++                                                +                                          +                                           +size of ORC read or streaming operators. Higher value will    +
++                                                +                                          +                                           +allow bigger chunks to be processed but will decrease         +
++                                                +                                          +                                           +concurrency level.                                            +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++      ``hive.orc.max-merge-`` ``distance``      +          ``String`` (data size)          +                 ``1 MB``                  +Serves as default value for ``orc_max_merge_distance``        +
++                                                +                                          +                                           +session property. Defines maximum size of gap between two     +
++                                                +                                          +                                           +reads to merge into a single read. The reads may be merged if +
++                                                +                                          +                                           +distance between requested data ranges in data source is      +
++                                                +                                          +                                           +smaller or equal to this value.                               +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++      ``hive.orc.stream-`` ``buffer-size``      +          ``String`` (data size)          +                 ``8 MB``                  +*Unused*                                                      +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++ ``hive.parquet-optimized-`` ``reader.enabled`` +               ``Boolean``                +                 ``false``                 +*Deprecated* Enables number of reader improvements introduced +
```

Comment:
This property was introduced in 0.138 (https://prestodb.io/docs/current/release/release-0.138.html) and is already deprecated? In favor of what?